integrations/pgvector/tests/test_delete_table.py

```diff
@@ -0,0 +1,20 @@
+import pytest
```

Comment:
Let's put these tests in haystack-core-integrations/integrations/pgvector/tests
/test_document_store.py